ntc_templates/templates/huawei_vrp_display_bgp_peer.textfsm

```diff
@@ -0,0 +1,18 @@
+Value PEER ([\w\.:]+)
```

Comment:
I'm suggesting we can simplify the `PEER` regex.
I have tested locally and this functions.

@brian-s-cao You can accept my suggestion by clicking the [commit suggestion](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request#applying-suggested-changes) button. Thank you!

```suggestion
Value PEER (\S+)
```